auto_install/install.sh

```diff
@@ -296,6 +311,38 @@ distroCheck(){
 				;;
 			esac
 		;;
+		Alpine)
+		PKG_MANAGER='apk'
+		UPDATE_PKG_CACHE="${PKG_MANAGER} update"
+		PKG_INSTALL="${PKG_MANAGER} --no-cache add"
+		PKG_COUNT="${PKG_MANAGER} --no-cache upgrade -s | grep -sEe ' [0-9]+ packages' | sed -E -e 's/.* ([0-9]+) packages/\1/' || true"
+		CHECK_PKG_INSTALLED="${PKG_MANAGER} --no-cache info -e"
+
+		# install grepcidr manually, if not present
+		if ! command -v grepcidr &> /dev/null; then
+			## install dependencies
+			${PKG_INSTALL} build-base make curl tar
+
+			## download binaeries
+			curl -fLo master.tar.gz https://github.com/pivpn/grepcidr/archive/master.tar.gz
+			tar -xzf master.tar.gz
+
+			## personalize binaries
+			sed -i -E -e 's/^PREFIX\=.*/PREFIX\=\/usr\nCC\=gcc/' grepcidr-master/Makefile
+
+			## install
+			(
+				cd grepcidr-master || exit
+
+				make
+				make install
+			)
+
+			## remove useless files
+			rm master.tar.gz
+			rm -rf grepcidr-master
```

Comment:
Installing grepcidr doesn't belong to `distroCheck()`.

Moreover it makes the script error out when being run without root

```
$ ./install.sh 
ERROR: Unable to lock database: Permission denied
ERROR: Failed to open apk database: Permission denied
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 16558    0 16558    0     0   2427      0 --:--:-- --:--:-- --:--:--  2427
./install.sh: line 337: make: command not found
./install.sh: line 338: make: command not found
[...]
```

`preconfigurePackages()` should be more appropriate.